editions/webdav/tiddlers/TiddlerListTemplate.tid

```diff
@@ -0,0 +1,3 @@
+title: TiddlerListTemplate
```

Comment:
I think, new templates should be part of the plugins, **if** the are essential for plugin function. If they are docs only, edition is OK.

The edition shouldn't be needed, if a user wants to use new functionality

Reply:
I see. We have to have a closer look here! ...